common/containers/Tabs/Contracts/components/Interact/components/InteractForm/index.tsx

```diff
@@ -76,6 +77,37 @@ class InteractForm extends Component<Props, State> {
     return contracts && contracts.length;
   };
 
+  public componentDidMount = () => {
+    setTimeout(
+      function() {
+        this.contractSetup();
+        const item = document.getElementById('accessButton');
+        if (item) {
+          item.click();
+        }
+      }.bind(this),
+      100
+    );
+  };
+
+  public contractSetup = () => {
+    const { contracts } = this.props;
+    // get contract name from url params
+    const contractName = queryString.parse(location.search).name;
+    if (this.isContractsValid()) {
+      contracts.map(con => {
+        const addr = con.address ? `(${con.address.substr(0, 10)}...)` : '';
+        // if contract name from Url exists, then load this contract
+        if (con.name === contractName) {
```

Comment:
Can you make a modification here to do the comparison as both strings in lowercase for a UX factor (ie: `MyContract` contract name won't match `mycontract` query string value)

Reply:
Good idea! Thanks. Did just that. Still not working on a live version even tho it works on a localhost so making a bunch of console.log to understand what's happing